runtime/rasdump/rasdump_internal.h

```diff
@@ -126,6 +126,9 @@ void setAllocationThreshold(J9VMThread *vmThread, UDATA min, UDATA max);
 
 #define J9RAS_DUMP_EXCEPTION_EVENT_GROUP (J9RAS_DUMP_ON_EXCEPTION_THROW | J9RAS_DUMP_ON_EXCEPTION_SYSTHROW | J9RAS_DUMP_ON_EXCEPTION_CATCH | J9RAS_DUMP_ON_EXCEPTION_DESCRIBE)
 
+#define J9PRINT_TO_STDOUT "/STDOUT/"
```

Comment:
How about J9RAS_PRINT_TO...

Reply:
I Shall change the macros to : 
J9RAS_PRINT_TO_STDOUT
J9RAS_PRINT_TO_STDERR

Is the location of the macro fine?

Reply:
@keithc-ca are you good with the location?

Reply:
I'm fine with the location, but I'd suggest the names be `J9RAS_STDOUT_NAME` and `J9RAS_STDERR_NAME`.
I'm not a fan of the slashes (`/`) in values; it may be better if the names couldn't be confused for path names (e.g. `[stdout]` or some other characters that are illegal (at least on Windows)).

Reply:
Okay I shall change the macros to :
J9RAS_STDOUT_NAME
J9RAS_STDERR_NAME